content/en/synthetics/browser_tests/actions.md

```diff
@@ -13,106 +13,139 @@ further_reading:
 
-Asserts that some specific text is **NOT** present on the current page.
+Create this assertion step to have your browser test confirm that the text you specified in the `Value` field is **not** present on the current page being recorded.
+
+#### Test that some text is present on the active page
+
+Create this assertion step to have your browser test confirm that the text you specified in the `Value` field is present on the current page being recorded.
 
 #### Test the content of the URL of the active page
 
-Takes the URL of the last page that was interacted with, then asserts whether a specific value (such as `string`, `number`, or `regex`) is present within it.
+Create this assertion step to have your browser test verify that the URL of the last page that was interacted with contains a value you specified.
+
+You can test for a value in the URL such as `string`, `number`, or `regex`.
 
-#### Test that an element is present on the active page
+{{% /tab %}}
 
-Asserts that an element (such as a specific `span`, `div`, `h`, or `a`) is present on the current page.
+{{% tab "Special Assertions" %}}
 
 #### Test that an email was received
 
-Asserts that an email was sent and whether specific values (such as `string`, `number`, or `regex`) are present within the email subject or body. This assertion leverages [email variables][4], you consequently first need to create an email variable to be able to use a `Test that an email was received` assertion.
+Create this assertion step to have your browser test confirm that your application's email mechanisms are working and verify that the values you specified such as `string`, `number`, or `regex` are present in the email subject or body. 
 
-#### Test your UI with custom JavaScript
+For more information, see [Email Validation with Browser Tests][1].
 
-Test a custom assertion on the active page using your own JavaScript code.
+#### Test your UI with custom JavaScript
 
-**Note**: JavaScript assertions support both synchronous and asynchronous code.
+Create this assertion step to test a custom assertion on the active page using your JavaScript code. JavaScript assertions support both synchronous and asynchronous code. Because browser tests load external JavaScript by adding the script to the page, they only work if your website accepts external JavaScript.
 
-The JavaScript assertion function comes with the following parameters and requires a return statement.
+The JavaScript assertion function contains the following parameters and requires a return statement.
 
-* The `return` (mandatory) statement should reflect the condition the assertion needs to meet for your browser test step to be successful. Any type can be returned, but the value is automatically casted as a boolean.
+* The `return` (mandatory) statement reflects the condition the assertion needs to meet in order for your test step to succeed. Any type can be returned, but the value is automatically casted as a boolean.
```

Comment:
```suggestion
* The `return` (mandatory) statement reflects the condition the assertion needs to meet for your test step to succeed. Any type can be returned, but the value is automatically cast as a boolean.
```
I think in this case, "casted" either should be "cast" or "cased". "Casted" is not a word unless I am missing product specific context here.